### PR TITLE
Add osu! UCI article

### PR DESCRIPTION
**Pull request by cl8n**
might add more info (about the events, involvement with AX, ???) but for now I just wanted to put its existence on the wiki so I can link to it. will do more in later PRs

---

### CODE REVIEW

File: wiki/Organisations/osu!_UCI/en.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- California
+- club
+- college
+- Irvine
+- university
+---
+
+# osu! UCI
+
+**osu! UCI** is an osu! club centered around playing and discussing the game at the University of California, Irvine (*UCI*). They host a variety of events, most notably including seasonal LAN tournaments and weekly meetups.
```
Comment (Joehuu) [inline wiki/Organisations/osu!_UCI/en.md]:
```suggestion
**osu! UCI** is an osu! club centred around playing and discussing the game at the University of California, Irvine (*UCI*). They host a variety of events, most notably including seasonal LAN tournaments and weekly meetups.
```

File: wiki/Organisations/osu!_UCI/en.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+- ![][flag_US] Kenny "[Kenny](https://osu.ppy.sh/users/1225459)" Yam (*President*)
+- ![][flag_US] Adrien "[R34pling](https://osu.ppy.sh/users/7662172)" Huynh Huu
+- ![][flag_US] Alex "[Riku on osu](https://osu.ppy.sh/users/3071659)" Liu
+- ![][flag_US] Jason "[jlin_02](https://osu.ppy.sh/users/13192092)" Lin
+- ![][flag_US] Jonathan "[ilalalayou](https://osu.ppy.sh/users/3144766)" Cheng
+- ![][flag_US] Julissa "[phamitsu](https://osu.ppy.sh/users/13045418)" Becerra
+- ![][flag_US] Megan "[migin](https://osu.ppy.sh/users/11118735)" Ikemoto
+- ![][flag_US] Nicholas "[_Breeze](https://osu.ppy.sh/users/9726582)" Budding
+- ![][flag_US] Shannon Chen
```
Comment (Joehuu) [inline wiki/Organisations/osu!_UCI/en.md]: No user link?

Comment (cl8n) [reply wiki/Organisations/osu!_UCI/en.md]: she doesn't have an account

File: wiki/Organisations/osu!_UCI/en.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- California
+- club
+- college
+- Irvine
+- university
+---
+
+# osu! UCI
+
+**osu! UCI** is an osu! club centred around playing and discussing the game at the University of California, Irvine (*UCI*). They host a variety of events, most notably including seasonal LAN tournaments and weekly meetups.
```
Comment (TPGPL) [inline wiki/Organisations/osu!_UCI/en.md]:
```suggestion
**osu! UCI** is an osu! club centred around playing and discussing the game at the University of California, Irvine (*UCI*). They host a variety of events, most notable including seasonal LAN tournaments and weekly meetups.
```

Comment (cl8n) [reply wiki/Organisations/osu!_UCI/en.md]: correct as is (it's an adverb)

**Pull request merged**